docs/content/doc/features/comparison.en-us.md

```diff
@@ -118,7 +118,7 @@ _Symbols used in table:_
 | LDAP user synchronization | ✓ | ✘ | ✓ | ✓ | ✓ | ✓ | ✓ |
 | OpenId Connect support | ✓ | ✘ | ✓ | ✓ | ✓ | ? | ✘ |
 | OAuth 2.0 integration (external authorization) | ✓ | ✘ | ⁄ | ✓ | ✓ | ? | ✓ |
-| Act as OAuth 2.0 provider | ✘ | ✘ | ✓ | ✓ | ✓ | ✓ | ✘ |
+| Act as OAuth 2.0 provider | [✓](https://github.com/go-gitea/gitea/pull/5378) | ✘ | ✓ | ✓ | ✓ | ✓ | ✘ |
```

Comment:
Why link to the PR rather than just the tick?

Reply:
@techknowlogick

Reply:
@adelowo ya, that was me. I figured as there was a PR that affected this I could get it in rather than making a PR just for a one lline change.

Reply:
In regards to why the link, we are starting to link to PRs where things are implemented so users can have more information about where the item was implemented.